Yoast/javascript
# Improve stemming Indonesian suffixes -kan and -an

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds check for words ending in -kan with stem ending in -k. If the stem ends in -k, only -an should be stemmed.
* [Yoast SEO Premium] Improves the recognition of words with stem ending -k which get suffix -an.

## Test instructions
This PR can be tested by following these steps:

* Make sure all current specs pass
* Add more examples to the specs and make sure they pass. To create the examples, take entries from the doNotStemK list in the Indonesian morphology data file (https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/180) and attach suffix -an in the end of the word . Also try combining the suffix with relevant prefixes (e.g. ber- + word + -an, per- + words with vowel/b beginning + -an ).
* In Indonesian Premium: Add words beginning with stem ending -k and suffix -an to the keyphrase and the text/title/meta description, also with the prefix combination above. See whether the forms are recognised.


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-431
